lib/service/feralfile_service.dart

```diff
@@ -144,11 +146,15 @@ abstract class FeralFileService {
 
   Future<Exhibition> getFeaturedExhibition();
 
-  Future<List<Artwork>> getExhibitionArtworks(String exhibitionId,
-      {bool withSeries = false});
+  Future<FeralFileListResponse<Artwork>> getExhibitionArtworks(
+      String exhibitionId,
+      {bool withSeries = false,
+      int offset = 0,
+      int limit = 300});
```

Comment:
define const for limit

Reply:
Should we remove the default value of parameter and use `limit ?? const` in function body. Use constant for this to easily change later if needed.